source/quick-start.txt

```diff
@@ -7,193 +7,182 @@ Quick Start
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 1
+   :depth: 2
    :class: singlecol
 
-This guide shows you how to create an application that uses the Node.js
-driver to connect to a MongoDB Atlas cluster.
+*Estimated time to complete this guide: 35 minutes.*
```

Comment:
Love the concept of estimated time to complete. Is there any specific way you've come to this number?

Reply:
I summed the time for the pages in the Atlas Getting Started Guide (29 minutes) and then added some time to account for the rest of the sections (install node/npm/deps and run the code sample).

Reply:
I like the idea too but it might be discouraging to see that as the first line of a "quick" start. There's not much we can do to decrease this number outside of actual improvements to products we don't own. Do we need to include it?

Reply:
I can remove it if you want. I thought it was helpful to set expectations, but can see how surfacing an estimated time that's more than a few minutes can cause a user to bounce.

Reply:
Thanks